service/src/main/resources/db/changesets/20251010_add_pipeline_variable_display_name_and_description.yaml

```diff
@@ -0,0 +1,112 @@
+# add display_name and description for pipeline variables
+
+databaseChangeLog:
+  -  changeSet:
+       id:  add display_name and description for pipeline variables and set values for array_imputation v1
+       author:  mma
+       changes:
+         # add display_name and description columns to pipeline_input_definitions table
+         - addColumn:
+             columns:
+               - column:
+                   name: display_name
+                   type: text
+                   constraints:
+                     nullable: true
+             tableName: pipeline_input_definitions
+         -  addColumn:
+              columns:
+                -  column:
+                     name:  description
+                     type:  text
+                     constraints:
+                       nullable: true
+              tableName:  pipeline_input_definitions
+
+         # add display_name and description columns to pipeline_output_definitions table
+         - addColumn:
+             columns:
+               - column:
+                   name: display_name
+                   type: text
+                   constraints:
+                     nullable: true
+             tableName: pipeline_output_definitions
+         - addColumn:
+             columns:
+               - column:
+                   name: description
+                   type: text
+                   constraints:
+                     nullable: true
+             tableName: pipeline_output_definitions
+
+         # set display_name and description values for user-provided array_imputation v1 inputs
```

Comment:
TOL and certainly out of scope for this PR: in the future would we want to be able to set these types of values via admin api instead of using liquibase?